AdobeStockAssetApi/Api/Data/UserProfileInterface.php

```diff
@@ -26,6 +26,7 @@ interface UserProfileInterface
     const REFRESH_TOKEN = 'refresh_token';
     const CREATED_AT = 'created_at';
     const UPDATED_AT = 'updated_at';
+    const EXPIRES_AT = 'expires_at';
```

Comment:
I think it will be better to rename the field to `access_token_expires_at` not to confuse with user profile expiration :)